.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,30 @@
+<!--- Provide a general summary of your changes in the Title above -->
+
+## Description
+<!--- Describe your changes in detail -->
+
+## Motivation and Context
+<!--- Why is this change required? What problem does it solve? -->
+<!--- If it fixes an open issue, please link to the issue here. -->
+
+## How Has This Been Tested?
+<!--- Please describe in detail how you tested your changes. -->
+<!--- Include details of your testing environment, and the tests you ran to -->
+<!--- see how your change affects other areas of the code, etc. -->
+
+## Screenshots (if appropriate):
+<!--- Drag-and-drop any relevant screenshots here, if applicable. -->
+
+## Types of changes
+<!--- What types of changes does your code introduce? Put an `x` in all the boxes that apply: -->
+- [ ] Bug fix (non-breaking change which fixes an issue)
+- [ ] New feature (non-breaking change which adds functionality)
+- [ ] Breaking change (fix or feature that would cause existing functionality to change)
```

Comment:
Task lists add up in the PR summary screen as a progress bar. Maybe use a regular bulleted list here to not overload it with IMO meaningless "TODOs"?

<img width="816" alt="screen shot 2016-09-10 at 14 29 32" src="https://cloud.githubusercontent.com/assets/2153375/18411078/07a43b9c-7768-11e6-8014-e7120e6cfb2a.png">

I think using GitHub task lists for the checklist section below makes sense since they're post-merge steps.